develop/parachain-devs/interoperability/hrmp-channels/para-to-para.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Opening HRMP Channels Between Parachains
+description: Learn how to open HRMP channels between parachains on Polkadot. Discover the step-by-step process for establishing uni- and bidirectional communication.
+---
+
+# Opening HRMP Channels Between Parachains
+
+## Introduction
+
+For establishing communication channels between parachains on the Polkadot network using the Horizontal Relay-routed Message Passing (HRMP) protocol, the following steps are required:
+
+1. **Channel request** - the parachain that wants to open an HRMP channel must make a request to the parachain it wishes to have an open channel with
+2. **Channel acceptance** - the other parachain must then accept this request to complete the channel establishment
+
+This process results in a unidirectional HRMP channel, where messages can flow in only one direction between the two parachains.
+
+An additional HRMP channel must be established in the opposite direction to enable bidirectional communication. This requires repeating the request and acceptance process but with the parachains reversing their roles.
+
+Once both unidirectional channels are established, the parachains can send messages back and forth freely through the bidirectional HRMP communication channel.
+
+## Prerequisites
+
+Before proceeding, ensure you meet the following requirements:
+
+- Blockchain network with a relay chain and at least two connected parachains
```

Comment:
Maybe seems a little odd to say "Blockchain network", maybe just network? Or Polkadot SDK-based network (sorry for the very picky nit!)